template/src/ontology/Makefile.jinja2

```diff
@@ -196,7 +196,7 @@ $(ROBOT_PLUGINS_DIRECTORY)/{{ plugin.name }}.jar:
 MAIN_PRODUCTS = $(sort $(foreach r,$(RELEASE_ARTEFACTS), $(r)) $(ONT))
 MAIN_GZIPPED  = {% if project.gzip_main %}$(foreach f,$(FORMATS), $(ONT).$(f).gz){% endif %}
 MAIN_FILES    = $(foreach n,$(MAIN_PRODUCTS), $(foreach f,$(FORMATS), $(n).$(f))) $(MAIN_GZIPPED)
-SRCMERGED     = $(TMPDIR)/merged-$(SRC)
+SRCMERGED     = $(TMPDIR)/merged-$(ONT)-edit.ofn
```

Comment:
Not sure how this change works:

```
$(SRCMERGED): $(EDIT_PREPROCESSED) $(OTHER_SRC)
	$(ROBOT) remove --input $< --select imports --trim false \
		merge  $(patsubst %, -i %, $(OTHER_SRC)) -o $@

$(EDIT_PREPROCESSED): $(SRC)
	$(ROBOT) convert --input $< --format ofn --output $@
```

EDIT_PREPROCESSED by default always converts to functional, regardless of the file ending, and the way SRCMERGED is defined, it should not really change the serialisation unless someone manually overwrites the goal (I mean, there is not `convert` step in there). Dont you have to at least at a convert -f ofn clause into the SRCMERGED goal?

Reply:
Notice how the command that produces `$(SRCMERGED)` does not have an explicit `convert -f...` bit. Therefore, the format of the resulting file is solely determined by the extension of the file given to the last `-o` option.

With `$(SRCMERGED)` defined as `$(TMPDIR)/merged-$(SRC)`, that extension will be the same as the extension of the -edit file. So if the -edit file is in OBO, `$(SRCMERGED)` is in OBO as well -- regardless of whatever format is used for `$(EDIT_PREPROCESSED).

Reply:
For the avoidance of doubt: ROBOT does _not_ automatically preserve the input format. Just because the input file of a pipeline is in OFN, it doesn’t mean the output file will be in OFN as well without an explicit `convert`. The format of the output file is always determined by the extension found in the argument to the `-o` option (or of course by the `-f` option, in the specific case of the `convert` command`).

Reply:
> For the avoidance of doubt: ROBOT does not automatically preserve the input format. Just because the input file of a pipeline is in OFN, it doesn’t mean the output file will be in OFN as well without an explicit convert. The format of the output file is always determined by the extension found in the argument to the -o option (or of course by the -f option, in the specific case of the convert command).

lol I didn't know this! I guess I never paused to think this might be the case.